backend/Techfesia2019/Techfesia2019/urls.py

```diff
@@ -24,6 +24,10 @@
     path('admin/', admin.site.urls),
     path('docs', schema_view),
     path('auth/', include("registration.urls")),
+    path('users/<str:username>/invitations/', include('event_registrations.invitation_urls')),
```

Comment:
Instead of having multiple routes for users, can you try to combine into one,
"""
path('users/', include([
                            # both the routes
                           ]).

Not sure if it will work. But looks cleaner.

Also where are the events routes?
`/events/`

Reply:
I checked from the beginning, there wasn't any from the start.

Reply:
Will try to combine the `/users/` routes

Reply:
> I checked from the beginning, there wasn't any from the start.

Leave it then. Events team can take care